action/action-dsl/src/main/groovy/com/powsybl/action/dsl/ActionDslLoader.groovy

```diff
@@ -236,6 +240,8 @@ class ActionDslLoader extends DslLoader {
 
             shell.evaluate(dslSrc)
 
+            actionsId.stream().forEach({ a -> rulesDb.getAction(a) })
```

Comment:
actionsId.forEach() : you don't need to get a stream to iterate. Please move this line in a private method with an explicit name because if we read this code in few monthes, we'll never remember that the getAction() method throws an exception if the action doesn't exist.